site/content/docs/4.3/layout/grid.md

```diff
@@ -365,6 +364,160 @@ You can also use the accompanying Sass mixin, `row-cols()`:
 }
 {{< /highlight >}}
 
+## Gutters
+
+Gutters can be responsively adjusted by breakpoint-specific gutter classes in as well horizontal, vertical and both directions. By default, `.row`s have a horizontal gutter of `1.5rem`. Removing this default gutter is possible by adding the `.g-0` class.
```

Comment:
> Removing this default gutter is possible by adding the `.g-0` class.

Very minor thing, but this description is explained in detail below, so it feels redundant here.

Reply:
Changed the copy below